frontend/src/lib/components/SelectBox.tsx

```diff
@@ -5,7 +5,7 @@ import { List } from 'antd'
 import { DownOutlined, RightOutlined } from '@ant-design/icons'
 import { ActionType, CohortType } from '~/types'
 import { selectBoxLogic } from 'lib/logic/selectBoxLogic'
-import './SelectBox.scss'
+// import './SelectBox.scss' // https://github.com/PostHog/posthog/issues/4524
```

Comment:
It might be needed to keep this around for shared dashboards, not sure.

Reply:
Avoiding importing the SCSS twice (once in global.scss) — which may or may not cause problems.